pallets/ocex/src/lib.rs

```diff
@@ -159,6 +157,7 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
+		FailedToConvertDecimaltoBalance,
```

Comment:
please add document desctiption - it's converted into message on polkadot/UI side